tests/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+def pytest_generate_tests(metafunc):
+    if "config_name" in metafunc.fixturenames:
```

Comment:
Do we really need this level of magic? I'm not even sure what it does precisely...